.github/workflows/ci.yaml

```diff
@@ -188,6 +188,7 @@ jobs:
           | yq w -d1 - spec.values.image.registry "${{ steps.login-ecr.outputs.registry }}" \
           | yq w -d1 - spec.values.image.repository "$DEPLOYMENT_NAME" \
           | yq w -d1 - spec.values.image.tag "$REF_DASHED-$GITHUB_SHA" \
+          | yq w -d1 - spec.values.serviceAccount.iamRole "arn:aws:iam::${{ steps.setup-aws.outputs.aws-account-id }}:role/ui-role-${{ matrix.environment }}" \
```

Comment:
why do we need to add that role during the CI build?

Reply:
Because now that the UI is server-side rendered it needs an IAM role to access AWS server-side for getting Cognito info, just like for any other service. We now have components of the UI running on the server instead of everything running by the user.